src/util-decode-der-get.c

```diff
@@ -128,6 +191,57 @@ const Asn1Generic * Asn1DerGet(const Asn1Generic *top, const uint8_t *seq_index,
     return node;
 }
 
+int Asn1DerGetValidity(const Asn1Generic *cert, time_t *not_before,
+                       time_t *not_after, uint32_t *errcode)
+{
+    const Asn1Generic *node, *it;
+    int rc = -1;
+
+    if (errcode)
```

Comment:
You are right. 

In the other ASN1 functions "errcode" is set to 0 right before returning when no error is encountered. It probably makes most sense to just remove it, since "errcode" is not used any way when there is no error.